templates/marketplace/dsm-mp.template

```diff
@@ -368,6 +368,7 @@ Resources:
               - ec2:DescribeAvailabilityZones
               - ec2:DescribeSecurityGroups
               - iam:ListAccountAliases
+              - sts:AssumeRole
```

Comment:
What is this permission required for? Are you creating temporary credentials for any role *? Can it be scoped down?

Reply:
The permission is required for using a cross account role to add an additional account into the DSM to be protected. We can't really scope it down because we don't know ahead of time the accounts or ARNs the customer is going to add.

https://help.deepsecurity.trendmicro.com/10/0/Add-Computers/add-aws.html?Highlight=add%20aws%20account#Add

Reply:
Ah I see.